src/java/org/apache/nutch/parse/ParseOutputFormat.java

```diff
@@ -338,6 +340,9 @@ public static String filterNormalize(String fromUrl, String toUrl,
       } catch (MalformedURLException e1) {
         return null; // skip it
       }
+
+
+
       if ("bydomain".equalsIgnoreCase(ignoreExternalLinksMode)) {
         String toDomain = URLUtil.getDomainName(targetURL).toLowerCase();
         if (toDomain == null || !toDomain.equals(origin)) {
```

Comment:
Shouldn't this case also be covered (db.ignore.external.links == true and db.ignore.external.links.mode == byDomain)?